test/dynamo/test_python_dispatcher.py

```diff
@@ -0,0 +1,59 @@
+# Owner(s): ["module: dynamo"]
+import unittest
+
+import torch
+import torch._dynamo.test_case
+from torch._dispatch.python import enable_python_dispatcher
+from torch._dynamo.testing import CompileCounter
+from torch.testing._internal.common_cuda import TEST_CUDA
+
+
+class PythonDispatcherTests(torch._dynamo.test_case.TestCase):
+    def test_dispatch_key(self):
+        @torch.compile(backend="eager", fullgraph=True)
+        def fn(x):
+            key_set = torch._C._dispatch_tls_local_include_set()
+            key_set = key_set | torch._C._dispatch_keys(x)
+            key_set = key_set - torch._C._dispatch_tls_local_exclude_set()
+            if key_set.highestPriorityTypeId() == torch.DispatchKey.PythonDispatcher:
+                return torch.sin(x + 1)
+            else:
+                return torch.sin(x - 1)
```

Comment:
@yanboliang what does the Dynamo graph look like here? I'm trying to figure out if there are DispatchKeySet operations in the graph. (an expecttest with EagerAndRecordGraphs() would help)

Reply:
I don't think these should go into the graph but some of the code looked suspicious to me

Reply:
You are right, the previous version is incorrect, updated.